File: WWF/Hands on labs 3.0/Lab02/Completed/Exercise1/HelloWorldActivityLibrary/HelloWorldActivity.cs
//--------------------------------------------------------------------------------
// This file is a "Sample" as from Windows Workflow Foundation
// Hands on Labs RC
//
// Copyright (c) Microsoft Corporation. All rights reserved.
//
// This source code is intended only as a supplement to Microsoft
// Development Tools and/or on-line documentation.  See these other
// materials for detailed information regarding Microsoft code samples.
// 
// THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
// KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
// IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
// PARTICULAR PURPOSE.
//--------------------------------------------------------------------------------

using System;
using System.ComponentModel;
using System.ComponentModel.Design;
using System.Collections;
using System.Drawing;
using System.Workflow.ComponentModel.Compiler;
using System.Workflow.ComponentModel.Serialization;
using System.Workflow.ComponentModel;
using System.Workflow.ComponentModel.Design;
using System.Workflow.Runtime;
using System.Workflow.Activities;
using System.Workflow.Activities.Rules;

namespace HelloWorldActivityLibrary
{
	public partial class HelloWorldActivity: SequenceActivity
	{
		public HelloWorldActivity()
		{
			InitializeComponent();
		}

		private void writeHello_ExecuteCode(object sender, EventArgs e)
		{
			Console.Write("Hello, ");
		}

		private void writeWorld_ExecuteCode(object sender, EventArgs e)
		{
			Console.WriteLine("World");
		}
	}
}

File: WWF/Hands on labs 3.0/VB/Lab02/Completed/Exercise1/CompositeActivitySample/HelloWorldActivityLibrary/HelloWorldActivity.vb
'--------------------------------------------------------------------------------
' This file is a "Sample" as from Windows Workflow Foundation
' Hands on Labs RC
'
' Copyright (c) Microsoft Corporation. All rights reserved.
'
' This source code is intended only as a supplement to Microsoft
' Development Tools and/or on-line documentation.  See these other
' materials for detailed information regarding Microsoft code samples.
' 
' THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
' KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
' IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
' PARTICULAR PURPOSE.
'--------------------------------------------------------------------------------

'NOTE: When changing the namespace; please update XmlnsDefinitionAttribute in AssemblyInfo.vb
Public Class HelloWorldActivity
    Inherits SequenceActivity
    Public Sub New()
        MyBase.New()
        InitializeComponent()
    End Sub

    Private Sub writeHello_ExecuteCode(ByVal sender As System.Object, ByVal e As System.EventArgs)
        Console.Write("Hello, ")
    End Sub

    Private Sub writeWorld_ExecuteCode(ByVal sender As System.Object, ByVal e As System.EventArgs)
        Console.WriteLine("World")
    End Sub
End Class